Week-9/todo-list/src/index.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
```

Comment:
Good practise is load the css as the last import statement. 

https://dev.to/coderamos/import-hierarchy-2deh